Sources/MuxPlayerSwift/FairPlay/FairPlaySessionManager.swift

```diff
@@ -230,13 +241,21 @@ class DefaultFairPlayStreamingSessionManager<
         offline _: Bool,
         completion requestCompletion: @escaping (Result<Data, Error>) -> Void
     ) {
-        let url = URL(
+        guard let url = URLComponents(
             playbackID: playbackID,
             drmToken: drmToken,
-            licenseDomain: String.makeLicenseDomain(
-                rootDomain: rootDomain
+            licenseHostSuffix: rootDomain
+        ).url else {
+            requestCompletion(
```

Comment:
Previously this was an error that'd be triggered by a failing force unwrap (`!`) in the URL constructor.

Reply:
🙌